fgrpc/grpcrunner_test.go

```diff
@@ -546,7 +557,7 @@ func Test_extractDialOptions(t *testing.T) {
 			if !reflect.DeepEqual(len(gotOut), tt.wantOutLen) {
 				t.Errorf("extractDialOptions() = %v, want %v", len(gotOut), tt.wantOutLen)
 			}
-			if !reflect.DeepEqual(tt.args.in, tt.wantMD) {
+			if !reflect.DeepEqual(sanitize(tt.args.in), tt.wantMD) {
```

Comment:
using the function that ends up being tested in the test isn't super great... it's tautological

Reply:
but removing sanitize (because the old extractDialOptions mutates tt.args.in) "works" for that purpose

Reply:
> using the function that ends up being tested in the test isn't super great... it's tautological

thanks a lot for the suggestion, but what do you mean by "the function that ends up being tested"? This test function just reuses the test cases to test "extractDialOptions" and "sanitize" (do you mean shouldn't test 2 functions at the same time in the same test function?

Reply:
ic, yes now that it's one function it's clearer I think